react/components/RequestOrganizationForm.tsx

```diff
@@ -255,203 +283,237 @@ const RequestOrganizationForm: FC = () => {
-                  label={translateMessage(messages.firstName)}
-                  value={formState.firstName}
-                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                    setFormState({
-                      ...formState,
-                      firstName: e.target.value,
-                    })
-                  }}
-                />
-              </div>
-              <div
-                className={`${handles.newOrganizationInput} mb5 flex flex-column`}
-              >
-                <Input
-                  size="large"
-                  label={translateMessage(messages.lastName)}
-                  value={formState.lastName}
-                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                    setFormState({
-                      ...formState,
-                      lastName: e.target.value,
-                    })
-                  }}
-                />
-              </div>
-              <div
-                className={`${handles.newOrganizationInput} mb5 flex flex-column`}
-              >
-                <Input
-                  size="large"
-                  label={translateMessage(messages.email)}
-                  value={formState.email}
-                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                    setFormState({
-                      ...formState,
-                      email: e.target.value,
-                    })
-                  }}
-                />
-              </div>
-            </PageBlock>
-            <PageBlock
-              variation="full"
-              title={translateMessage(messages.defaultCostCenter)}
-              subtitle={translateMessage(messages.defaultCostCenterHelpText)}
-            >
-              <div
-                className={`${handles.newOrganizationInput} mb5 flex flex-column`}
-              >
-                <Input
-                  autocomplete="off"
-                  size="large"
-                  label={translateMessage(messages.defaultCostCenterName)}
-                  value={formState.defaultCostCenterName}
-                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                    setFormState({
-                      ...formState,
-                      defaultCostCenterName: e.target.value,
-                    })
-                  }}
-                />
-              </div>
-              <div
-                className={`${handles.newOrganizationAddressForm} mb5 flex flex-column`}
-              >
-                <AddressRules
-                  country={addressState?.country?.value}
-                  shouldUseIOFetching
-                  useGeolocation={false}
+            {!isAuthenticated ? (
+              <PageBlock>
+                <div>
+                  <FormattedMessage id="store/b2b-organizations.not-authenticated" />
+                </div>
+              </PageBlock>
+            ) : formState.submitted &&
```

Comment:
Checking for `formState.submitted` here means that the user will always see the blank form if they come back to the page later (because `formState.submitted` has an initial value of `false`). We want the user to be able to see the status of their request by returning to this page. 

I think all you need to do is give `formState.submitted` an initial value of `true`, and then change it to `false` if the user clicks the "New Request" button. Nothing else should change the value.

Reply:
Sorry @arturmagalhaesjr , I was wrong when I said that "nothing else should change the value."

Scenario:
- User returns to the page after previously submitting an org request. The form is hidden and the status of their request is shown <---- this is working 👍 
- They want to submit a new request, so they click the New Request button. The form is displayed again <---- this is working 👍 
- They submit the new form. The form should now be hidden again and they should see the message about the status of their request <---- this is not working 👎 

I think the solution is to set `formState.submitted` to `true` when the form is submitted.